spec/api-browser-window-spec.js

```diff
@@ -1357,6 +1357,49 @@ describe('BrowserWindow module', () => {
     afterEach(() => { ipcMain.removeAllListeners('answer') })
 
     describe('"preload" option', () => {
+      const doesNotLeakSpec = (name, webPrefs) => {
+        it(name, async function () {
+          this.timeout(120000)
```

Comment:
does this really need an increased timeout? it's not doing anything that should take _2 minutes_.

Reply:
Oh no, my bad, I left this in from debugging